sale_stock_analytic/views/sale_order_view.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_order_form_inh_stock_analytic" model="ir.ui.view">
```

Comment:
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action

Use the following patterns, where <model_name> is the name of the model that the menu, view, etc. belongs to (e.g. for a res.users form view, the name would be res_users_view_form):

- For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...

Reply:
Realizada la mejora solicitada por @oihane , y subido el módulo. Eskerrik asko.